web/gui/dashboard.js

```diff
@@ -2458,22 +2494,23 @@ var NETDATA = window.NETDATA || {};
                     //console.log('DEBUG: ' + uuid.toString() + ' no conversion available for ' + units.toString() + ' to: ' + desired_units.toString());
                     switch_units_callback(units);
                     return function (value) { return value; };
-                }
-                else if(typeof this.convertibleUnits[units][desired_units] !== 'undefined') {
+                } else if (typeof this.convertibleUnits[units][desired_units] !== 'undefined') {
                     switch_units_callback(desired_units);
                     return this.convertibleUnits[units][desired_units].convert;
-                }
-                else {
+                } else {
                     console.log('Units conversion from ' + units.toString() + ' to ' + desired_units.toString() + ' is not supported.');
                     switch_units_callback(units);
-                    return function (value) { return value; };
+                    return function (value) { 
+                        return value; 
+                    };
                 }
-           }
-           else {
+           } else {
                 // hm... did we forget to implement the new type?
-                console.log('Unmatched unit conversion method for units ' + units.toString());
+                console.log(`Unmatched unit conversion method for units ${units.toString()}`);
```

Comment:
hm... I didn't know this can be done...

Reply:
Yes, javascript supports interpolation (it's faster too ;-))